public/apps/configuration/panels/auth-view/authentication-sequence-panel.tsx

```diff
@@ -14,16 +14,113 @@
  */
 
 import React from 'react';
+import { EuiInMemoryTable } from '@elastic/eui';
+import { _ } from 'lodash';
```

Comment:
Suggestion from @mihirsoni that import the function directly rather than import the whole package. My PR follows that style, better keep consistent.

Reply:
Sure. I get the idea of avoiding importing *. Originally just want to use "_." to distinguish those functions from local functions.